core/src/main/scala/org/elasticmq/SQSLimits.scala

```diff
@@ -0,0 +1,127 @@
+package org.elasticmq
+
+import scala.collection.JavaConverters._
+import scala.util.Try
+
+sealed trait SQSLimits {
```

Comment:
probably doesn't have to be sealed, very unlikely but we could without problems accept other implementations, no?

Reply:
There is possibility that there will be another implementations so I have "unsealed" it